hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -136,6 +136,14 @@ public HoodieTimeline filterPendingCompactionTimeline() {
         instants.stream().filter(s -> s.getAction().equals(HoodieTimeline.COMPACTION_ACTION) && !s.isCompleted()), details);
   }
 
+  @Override
+  public HoodieTimeline filterPendingCompactionAndClusteringTimeline() {
```

Comment:
```suggestion
  public HoodieTimeline filterPendingCompactionAndReplaceTimeline() {
```

insert overwrite, partition delete or clustering all result in replace commit

Reply:
Sure thing, changed.